src/scales/scale.linear.js

```diff
@@ -30,6 +30,8 @@ module.exports = function(Chart) {
 				return isHorizontal ? meta.xAxisID === me.id : meta.yAxisID === me.id;
 			}
 
+			me.minBarLength = opts.minBarLength;
```

Comment:
I would not expose this option as a public scale member but instead directly access `scale.options.minBarLength` from the bar controller as we do for other bar specific options.